electricitymap/contrib/config/__init__.py

```diff
@@ -38,6 +45,26 @@
         if zone_name_1 not in ZONE_NEIGHBOURS:
             ZONE_NEIGHBOURS[zone_name_1] = set()
         ZONE_NEIGHBOURS[zone_name_1].add(zone_name_2)
+
+# Find neighbors to subzones and add them to parent zone.
+for zone in ZONES_CONFIG.keys():
+    subzones = ZONES_CONFIG[zone].get("subZoneNames", [])
+    if not subzones:
+        continue
+
+    for subzone in subzones:
+        for subzone_neighbor in ZONE_NEIGHBOURS[subzone]:
+            if subzone_neighbor in subzones:
+                # ignore the neighbours that are within the zone
+                continue
+
+            if zone not in ZONE_NEIGHBOURS:
+                ZONE_NEIGHBOURS[zone] = set()
+
+            # If neighbor zone is a subzone, we add the exchange to the parent zone
```

Comment:
Image having `A-1`, `A-2`, `A-3`, `B-1`, `B-2`, `C` zones with exchanges between `A-1->A-2`, `A-2->B-1` and `A-2->C`. What would be the resulting neighbour relations?

Reply:
`A->B` and `A->C`